engine/apps/public_api/views/schedules.py

```diff
@@ -41,6 +41,9 @@ class OnCallScheduleChannelView(RateLimitHeadersMixin, UpdateSerializerMixin, Mo
     filter_backends = (filters.DjangoFilterBackend,)
     filterset_class = ByTeamFilter
 
+    # self.get_object() is not used in export action because ScheduleExportAuthentication is used
+    extra_actions_ignore_no_get_object = ["export"]
```

Comment:
how is this attribute consumed?

Reply:
ohh nvm, I see it's used in the new unit test [here](https://github.com/grafana/oncall/pull/2120/files#diff-440b76eaffde0806552596d9618b211d29e14a2cbb183a5065285094ae0c0d69R48-R81). Nice! 👍